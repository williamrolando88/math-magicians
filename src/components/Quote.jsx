import React from 'react';

const Quote = () => (
  <section>
    <h2 className="text-2xl">Lets get you inspired!</h2>
    <blockquote className="text-xl mt-4">
      Mathematics is not about numbers, equations, computations or algorithms,
      is about understanding. - William Paul Thurston
    </blockquote>
  </section>
);

export default Quote;
