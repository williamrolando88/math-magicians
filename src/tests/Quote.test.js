import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

test('should render Quote', () => {
  expect(renderer.create(<Quote />)).toMatchSnapshot();
});
