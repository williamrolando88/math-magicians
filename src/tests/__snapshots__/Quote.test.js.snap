// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Quote 1`] = `
<section>
  <h2
    className="text-2xl"
  >
    Lets get you inspired!
  </h2>
  <blockquote
    className="text-xl mt-4"
  >
    Mathematics is not about numbers, equations, computations or algorithms, is about understanding. - William Paul Thurston
  </blockquote>
</section>
`;
